cloudflare-mixin/dashboards/cloudflare-worker-overview.libsonnet

```diff
@@ -0,0 +1,399 @@
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'ms',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local workerDurationQuantilesPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'cloudflare_worker_duration{job=~"$job", instance=~"$instance", script_name=~"$script_name"}',
+      datasource=promDatasource,
+      legendFormat='{{script_name}} - {{quantile}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Worker duration quantiles',
+  description: 'The P50, P75, P99, P999 quantiles for worker duration.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 10,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'linear',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 's',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local workerRequestsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(cloudflare_worker_requests_count{job=~"$job", instance=~"$instance", script_name=~"$script_name"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{script_name}}',
+      interval='1m',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Worker requests / $__interval',
```

Comment:
Is there any reason you decided to go with increase() here instead of rate? one thing that was brought to my attention by Emily recently is this really important tidbit of info 
<img width="460" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/ad9ea98b-7857-466a-8c1a-052af78234ff">

Reply:
Eh, no particular reason, happy to change it to rate.